### PR TITLE
Add docs for Discourse

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ekohl** [comment]
This only adds it to the index, but the actual doc is missing.

**Comment by ehelms** [comment]
Git is hard... fixed

File: docs/discourse.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## What's running?
+
+Discourse runs in Docker, so very little is running at the main host level. There are 2 containers running:
+
+1. The main Discourse app, containing Rails, Redis, Sidekiq, and PSQL. Listens on 80 and 443 and provides it's own Let's Encrypt setup
+2. A Postfix container listening on 25 which handles incoming mail and then passes it to Discourse for turning into posts
+
+## Locations
+
+All the work is done in `var/discourse`. In that directory you'll find:
```
Comment (ekohl) [inline docs/discourse.md:16]:
```suggestion
All the work is done in `/var/discourse`. In that directory you'll find:
```

File: docs/discourse.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<23>Dec 26 06:19:44 receive-mail[17867]: Recipient: reply+<token-redacted>`community.theforeman.org
+```
+
+### Discourse
+
+If for some reason you need access to the Rails instance itself, you can enter the container with `./launcher enter app`. That will put you at the root of the rails app, so you can immediately do things like `bundle exec rake -T`
+
+For basic app control, `start`, `stop`, and `restart` are available, of course.
+
+h4. restarting
```
Comment (ekohl) [inline docs/discourse.md:40]:
```suggestion
#### restarting
```

File: docs/discourse.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+If for some reason the Discourse WebUI upgrade process fails, it will direct you to do a CLI upgrade, which is usually `./launcher rebuild app` but on-screen notes are usually provided.
+
+## Disaster recovery
+
+There are two forms of backup:
+
+### Volume snapshots
+
+Every day "a script":https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/scaleway/files/manage_snapshots.rb runs on the Discourse VM which talks to the Scaleway Snapshot API, triggers a new snapshot, and then (on success) cleans the older images away. The most recent 3 snapshots are retained.
```
Comment (ekohl) [inline docs/discourse.md:53]:
```suggestion
Every day [a script](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/scaleway/files/manage_snapshots.rb) runs on the Discourse VM which talks to the Scaleway Snapshot API, triggers a new snapshot, and then (on success) cleans the older images away. The most recent 3 snapshots are retained.
```

**Review by ekohl — APPROVED** [review]

**Pull request merged**